x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -466,11 +493,28 @@ export class EndpointDocGenerator {
       },
       host: this.commonInfo.host,
       process: {
+        pid: options.pid ? options.pid : this.randomN(5000),
+        executable: `C:\\${processName}`,
+        args: `"C:\\${processName}" \\${this.randomString(3)}`,
+        code_signature: {
+          status: 'trusted',
+          subject_name: 'Microsoft',
+        },
+        hash: { md5: this.seededUUIDv4() },
         entity_id: options.entityID ? options.entityID : this.randomString(10),
-        parent: options.parentEntityID ? { entity_id: options.parentEntityID } : undefined,
-        name: options.processName ? options.processName : randomProcessName(),
+        parent: options.parentEntityID
+          ? {
+              entity_id: options.parentEntityID,
+              pid: options.parentPid ? options.parentPid : this.randomN(5000),
```

Comment:
same as above, I think `'parentPid' in options` would be a better check here.

Reply:
https://github.com/elastic/kibana/pull/69098/commits/074bfc3a9f98077730997168ca66913a308fc935